internal/common/common.go

```diff
@@ -165,6 +165,13 @@ func (b *BuildConfig) Build() error {
 		"--workdir=/workspace",
 		// Remove the container file system after the container exits.
 		"--rm",
+		// The host uid and gid are needed to give the host access rights.
+		// --env has to be in separate string, otherwise docker
+		// interprets "--env HOST_UID=%d" as single flag instead of flag with
```

Comment:
With the whitespace it makes sense that it would not work as it would be parsed as a single flag, but I think it should work with `=` (i.e. `--env=HOST_UID=%d`)?

Reply:
Might be. Do you think it is worth a try?